packages/cli/test/mocks/deployment.ts

```diff
@@ -170,6 +170,16 @@ function setupDeploymentEndpoints(): void {
     res.json({ builds });
   });
 
+  client.scenario.get('/now/deployments/:id/aliases', (req, res) => {
```

Comment:
We should probably treat the first path segment as a version as described in the docs: https://vercel.com/docs/rest-api/endpoints#list-deployment-aliases

```suggestion
  client.scenario.get('/:version/deployments/:id/aliases', (req, res) => {
```